### DISCUSSION TITLE
Changes to some of the files in react-components atoms are not getting reflected on UI

### DISCUSSION THREAD

**Discussion by PavanSydhala** [egovernments/Digit-Core#286] [Bugs]
When I make changes to some of the Files inside the react-components atoms, they are not getting reflected on UI even after running yarn and restarting the application

**Comment by jagankumar-egov** [comment]
Verify the module package.json and the version of the React components package. If they differ, ensure uniformity by updating both to the same version. Subsequently, execute a yarn install after the update.
